xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -1612,7 +1612,7 @@
   "List.Fields.Files": {
     "extends": "Item.Fields.Base",
     "items": { "type": "string",
-      "enum": [ "title", "artist", "albumartist", "genre", "year", "rating",
+      "enum": [ "title", "artist", "albumartist", "genre", "year", "rating", "userrating",
```

Comment:
Please append "userrating" to the end of the list of enum values after "albumduration" rather than insert.
While I totally get why you would place this next to "rating", but appending new values is the prefered approach.

Reply:
Done